test/lib/ufe/testAttribute.py

```diff
@@ -1051,18 +1174,46 @@ def testAttributeLock(self):
         self.assertTrue(attr.hasMetadata(ufe.Attribute.kLocked))
         md = attr.getMetadata(ufe.Attribute.kLocked)
         self.assertTrue(bool(md))
+        if self.getAttrSupportsUfe:
+            self.assertFalse(cmds.getAttr(getsetAttrPath, settable=True))
+            self.assertTrue(cmds.getAttr(getsetAttrPath, lock=True))
 
         # Test that we cannot modify the height again.
         currValue = attr.get()
-        with self.assertRaises(RuntimeError):
-            attr.set(currValue + 1.0)
+        self.assertRaises(RuntimeError, attr.set, currValue + 1.0)
         self.assertEqual(currValue, attr.get())
+        if self.setAttrSupportsUfe:
+            self.assertRaises(RuntimeError, cmds.setAttr, getsetAttrPath, currValue+1.0)
 
         # Test that we cannot set other Metadata again.
-        with self.assertRaises(RuntimeError):
-            attr.setMetadata('documentation', 'New Doc')
+        self.assertRaises(RuntimeError, attr.setMetadata, 'documentation', 'New Doc')
+
+    def testMayaGetAttr(self):
+        if not self.getAttrSupportsUfe:
+            self.skipTest('Maya getAttr command does not support Ufe')
```

Comment:
Specific getAttr test which will log message if skipped. Cannot use function decorator for reason indicated above.